data/sql/updates/pending_db_world/rev_1652689266458530042.sql

```diff
@@ -0,0 +1 @@
+UPDATE `creature_template` SET `flags_extra` = 257 WHERE (`entry` = 14834);
```

Comment:
Add flags, dont overwrite them please. Like this

```
`flags_extra`=`flags_extra`|flag1|flag2
```

You can read it better here https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits if you wish

Reply:
I was wondering why Nef did it like that. THanks for the info